vyper/compiler/utils.py

```diff
@@ -4,12 +4,11 @@
 
 
 def build_gas_estimates(func_ts: Dict[str, ContractFunctionT]) -> dict:
-    # note: `.gas_estimate` is added to ContractFunctionT._ir_info
-    # in vyper/semantics/types/function.py
```

Comment:
Probably keep this

Reply:
Yup, it's kept as the function docstring rather ;)

Reply:
Oh I mean keep it as it was, not quite a docstring, just a dev comment